src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
It seems like you are trying to link a SCSS file in your HTML. Browsers can't interpret SCSS, they only understand CSS. You need to compile your SCSS into CSS and then link the CSS file in your HTML.